### PR TITLE
Add kafkacat recipe.

### PR DESCRIPTION
**Pull request by rolando**

### CODE REVIEW

File: recipes/kafkacat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - toolchain   # [unix]
+    # Compiling with latest 0.9.4 breaks backwards ABI compatibility.
+    - librdkafka 0.9.2
+    - yajl
```
Comment (jakirkham) [inline recipes/kafkacat/meta.yaml]: Should `yajl` have some version constraints?

File: recipes/kafkacat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+  run:
+    - librdkafka >=0.9.2
+    - yajl
+
+test:
+
```
Comment (jakirkham) [inline recipes/kafkacat/meta.yaml]: nit: Please drop blank line here.

File: recipes/kafkacat/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - conda inspect objects {{ name }}   # [osx]
+
+about:
+  home: https://github.com/edenhill/kafkacat
+  license: BSD 2-Clause
+  license_file: LICENSE
+  summary: "Generic command line non-JVM Apache Kafka producer and consumer"
+  #description:
+  #doc_url:
+  #dev_url:
```
Comment (jakirkham) [inline recipes/kafkacat/meta.yaml]: Please drop comments (unless you plan to fill these out).

File: recipes/kafkacat/build.sh [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+export CFLAGS="${CFLAGS} -O3 -fPIC -I${PREFIX}/include"
+export CXXFLAGS="${CXXFLAGS} -O3 -fPIC -I${PREFIX}/include"
+export LDFLAGS="${LDFLAGS} -Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib"
+export CPPFLAGS="${CXXFLAGS}"
+
+# Configure
+./configure --prefix=$PREFIX
+
+# Build
+make -j$CPU_COUNT
```
Comment (jakirkham) [inline recipes/kafkacat/build.sh]: Do we have some sort of `make check` that we can run?

File: recipes/kafkacat/meta.yaml (hunk 2 above)
Comment (rolando) [inline recipes/kafkacat/meta.yaml]: +1

File: recipes/kafkacat/meta.yaml (hunk 3 above)
Comment (rolando) [inline recipes/kafkacat/meta.yaml]: oops

File: recipes/kafkacat/build.sh (hunk 4 above)
Comment (rolando) [inline recipes/kafkacat/build.sh]: There is no `make check` and no tests to run.

File: recipes/kafkacat/meta.yaml (hunk 1 above)
Comment (rolando) [inline recipes/kafkacat/meta.yaml]: There is only one release in `conda-forge` but probably a good idea.

**Comment by rolando** [comment]
@jakirkham recipe was updated.

**Comment by jakirkham** [comment]
Thanks @rolando.

**Pull request merged**